### ISSUE TITLE
Character probabilities

### ISSUE THREAD

**Issue by GeorgiosHajivassiliou** [NVIDIA/OpenSeq2Seq#492]
Hello NVIDIA, I am trying to access the probability associated with each character at each time step of the spectrogram, where can I get this from the code?

**Comment by borisgin** [comment]
Do you mean logits duirng inference? There is a flag infer_logits_to_pickle’: True in the decoder_params . Take a look at 
[text aligner tool](https://nvidia.github.io/OpenSeq2Seq/html/speech-recognition/speech-to-text-align.html)
